docs/workflow/blocks/form.rst

```diff
@@ -81,21 +91,22 @@ Here's an example of creating a form by manually writing fields and configuring
         }
     },
     "uiSchema": {}
-}
+  }
 
 Dynamic data and field titles
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-To generate fields form given data, the data needs to be in a format readable by the form block.
+To generate fields form given data, the data must be in a format that the form block can read.
```

Comment:
Presumably this should be '**from** given data'?

Reply:
done